### DISCUSSION TITLE
Rounded version of new Logo looks weird

### DISCUSSION THREAD

**Discussion by N3rdix** [home-assistant/frontend#17988] [Configuration] [answered]
I just noticed that the new HA logo and the rounded design in the frontend don’t like each other.
Not sure if this was already discussed somewhere else, but maybe there is some way to resize the HA logo for such frontend display purposes 

![image](https://github.com/home-assistant/frontend/assets/70015952/385fea9e-4064-4041-8df8-8656480e5968)
![image](https://github.com/home-assistant/frontend/assets/70015952/ca7cf2c1-6649-4e9f-9bc4-f606c4f15d03)

**Comment by frenck** [comment, marked as answer]
We've removed the rounding for the next release of Home Assistant.

I will look into replacing it with a version that works with rounding in the mean time.

../Frenck
